age/README.md

```diff
@@ -0,0 +1,66 @@
+# Age Smart Contract
+
+This is a simple smart contract project that demonstrates how to make a simple mutli-function smart contract using massa tools.
+
+## Installation
+To use this project, you'll need to have Node.js >= v16.0  and NPM installed on your system. You can download Node.js from the official website [here](https://nodejs.org/en/download).
+
+Once you have Node.js and NPM installed, you can clone the project from GitHub using the following command:
+
+```bash
+git clone git@github.com:massalabs/massa-sc-examples.git
+```
+
+After cloning the project, navigate to the project directory, install and update the required dependencies using NPM:
+
+```bash
+cd massa-sc-examples/helloworld
+npm install
+npm update @massalabs/as-transformer @massalabs/as-types @massalabs/massa-as-sdk @massalabs/massa-sc-compiler @massalabs/massa-sc-deployer @massalabs/massa-web3
```

Comment:
Why do you need to update ? This could brake the code if you get an incompatible version, no ?